mergin/client_pull.py

```diff
@@ -44,7 +44,15 @@ class DownloadJob:
     """
 
     def __init__(
-        self, project_path, total_size, version, update_tasks, download_queue_items, directory, mp, project_info
+        self,
+        project_path,
+        total_size,
+        version,
+        update_tasks,
+        download_queue_items,
+        directory: tempfile.TemporaryDirectory,
```

Comment:
I would suggest to rename `directory` to something else, to make it clear it is a temporary directory, not the project directory anymore

Reply:
renamed